src/main.rs

```diff
@@ -42,6 +50,41 @@ struct Cli {
     )]
     groups: Vec<String>,
 }
+pub struct EmitKVPLayer {
+    file_path: std::path::PathBuf,
+}
+
```

Comment:
Adding the KVP code directly in `main.rs`, the file is getting bigger and becoming a mix of different things.
Would it be like a good idea to split the KVP/tracing-related part into another file in the top level, for example `kvp.rs`?

Reply:
Completely agree. I've separated all the kvp/tracing logic so that it is split into its own file (we'll have tracing.rs for all tracing logic and kvp.rs for all HyperV KVP related things).